assets/docs/callmemehdi/models/AraBERT/1.md

```diff
@@ -28,6 +30,57 @@ NER (ANERcorp) | macro-F1   | 78.4  | 81.9
 QA (ARCD) | macro-F1   | 61.3  | 62.7
 
 
+## Usage
+
+In order to use this model, you need to use the arabert repository to preprocess your input.
+First, clone their repository using this command:
+```shell
+git clone https://github.com/aub-mind/arabert.git
+```
+Then, use it to preprocess your input, here's an example:
+
+```python
+from arabert.preprocess import ArabertPreprocessor
+
+model_name="bert-base-arabertv2"
+arabert_prep = ArabertPreprocessor(model_name=model_name)
+
+text = "كان الملك رجلا والملكة مرأة"
+arabert_prep.preprocess(text)
+```
+Note that you would need to install some external libraries:
+```shell
+pip install pyarabic
+pip install farasapy
+```
+After preprocessing the input, we can start tokenizing it:
+
+```python
+from transformers import AutoTokenizer
+tokenizer = AutoTokenizer.from_pretrained("aubmindlab/bert-base-arabertv2")
+
+tokenized_text = tokenizer.tokenize(preprocessed_text)
+
+input = tokenizer.convert_tokens_to_ids(tokenized_text)
+```
+After that, you have your input ready, so you just have to pass it through the model
+
+```python
+import tensorflow as tf
+
+outputs = model(tf.convert_to_tensor([input]))
+
+```
+
+We need to take the last hidden state with ``` outputs[0] ``` which has the following shape:
```

Comment:
It might be helpful to describe the outputs more. What else does the model output?